### PR TITLE
Cv 59

### PR DESCRIPTION
**Pull request by abinashkonersman**
Implemented Secrets in  main function(main.py) of WZDx code in GCP.

### CODE REVIEW

File: translator/GCP_cloud_function/cloud_function/icone_translator.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
+import xmltodict
```
Comment (payneBrandon) [inline translator/GCP_cloud_function/cloud_function/icone_translator.py:1]: This seems to be a previous version without your latest changes. Did you pull before creating this PR?

File: translator/GCP_cloud_function/cloud_function/main.py [hunk 2]
```diff
@@ -52,18 +52,17 @@ def parse_xml(xml_string):
 def get_wzdx_schema(schema_file_name):
   return json.loads(open(schema_file_name,'r').read())
 
-#this function retrieves username and password info from secrets .This will be implemented after correct secrets are implemented.
 def get_ftp_credentials():
   secret_client = secretmanager.SecretManagerServiceClient()
-  username_secret_name = os.environ['icone_ftp_username_secret_name']
-  password_secret_name = os.environ['icone_ftp_password_secret_name']
+  username_secret_name = os.environ['icone_ftp_username_secret_name']#"icone_ftp_username"
```
Comment (payneBrandon) [inline translator/GCP_cloud_function/cloud_function/main.py]: Can we remove these comments? They don't seem to be explaining or helping.

**Comment by payneBrandon** [comment]
In `main.py` the `from gcloud import storage` doesn't seem to be needed. Can we remove?

Same with `import shutil`

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Can we update the GCP README file to include details about what environment variables are expected and what secrets the app needs?

File: translator/GCP_cloud_function/cloud_function/main.py (hunk 2 above)
Comment (abinashkonersman) [reply translator/GCP_cloud_function/cloud_function/main.py]: I removed those comments and pushed the updated code.

**Pull request closed without merging**